docs/core/run-time-config/globalization.md

```diff
@@ -76,3 +76,14 @@ Project file:
 | - | - | - |
 | **runtimeconfig.json** | `Switch.System.Globalization.FormatJapaneseFirstYearAsANumber` | `false` - format as "Gannen"<br/>`true` - format as number |
 | **Environment variable** | N/A | N/A |
+
+## NLS
+
+- Determines whether .NET uses National Language Support (NLS) or International Components for Unicode (ICU) globalization APIs for Windows apps. .NET 5.0 and later versions use ICU globalization APIs by default on Windows 10.
+- Default: Use ICU globalization APIs (`false`).
+- For more information, see [Globalization APIs use ICU libraries on Windows](../compatibility/3.1-5.0.md#globalization-apis-use-icu-libraries-on-windows).
+
+| | Setting name | Values | Introduced |
```

Comment:
It seems like this table has a syntax error:

![image](https://user-images.githubusercontent.com/22899328/82269313-2d894800-9926-11ea-8648-25603fe1bdef.png)

Reply:
Oops, thanks.